act-rules-format.bs

```diff
@@ -368,29 +379,31 @@ Definitions {#definitions}
   <dd>
     <p>A resource or collection of resources that can be evaluated by an ACT Rule.</p>
     <div class=example>
+      <p><strong>Example:</strong></p>
       <ul>
         <li>An HTML page, including all embedded scripts, style and images<li>
         <li>An EPUB document</li>
         <li>A Vue component file</li>
       </ul>
     </div>
     <div class=note>
-      <p>Implementers using the [[EARL10-Schema]] can express the test subject on the [subject property](https://www.w3.org/TR/EARL10-Schema/#subject)</p>
+      <p><strong>Note</strong>: Implementers using the [[EARL10-Schema]] can express the test subject on the [subject property](https://www.w3.org/TR/EARL10-Schema/#subject)</p>
```

Comment:
I'm thinking it should say "**using** the [subject property]" rather than "on..."

Reply:
Used "with" instead.